base/uk.ac.stfc.isis.ibex.ui.graphing/src/uk/ac/stfc/isis/ibex/ui/graphing/websocketview/MatplotlibToolbar.java

```diff
@@ -0,0 +1,128 @@
+package uk.ac.stfc.isis.ibex.ui.graphing.websocketview;
+
+import java.util.Map;
+
+import org.eclipse.swt.SWT;
+import org.eclipse.swt.events.SelectionAdapter;
+import org.eclipse.swt.events.SelectionEvent;
+import org.eclipse.swt.graphics.Image;
+import org.eclipse.swt.layout.GridData;
+import org.eclipse.swt.widgets.Composite;
+import org.eclipse.swt.widgets.ToolBar;
+import org.eclipse.swt.widgets.ToolItem;
+import org.eclipse.wb.swt.ResourceManager;
+
+public class MatplotlibToolbar {
+	private final ToolBar toolBar;
+	
+	private final Map<String, String> buttonIcons = Map.of(
```

Comment:
```suggestion
	private static final Map<String, String> buttonIcons = Map.of(
```

We don't need a new instance of this map for each class instance